content/docs/tasks/telemetry/gateways/index.md

```diff
@@ -0,0 +1,632 @@
+        kind: VirtualService
+        metadata:
+          name: prometheus-vs
+          namespace: istio-system
+        spec:
+          hosts:
+          - "$TELEMETRY_DOMAIN"
+          gateways:
+          - prometheus-gateway
+          http:
+          - match:
+            - port: 15030
+            route:
+            - destination:
+                host: prometheus
+                port:
+                  number: 9090
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: DestinationRule
+        metadata:
+          name: prometheus
+          namespace: istio-system
+        spec:
+          host: prometheus
+          trafficPolicy:
+            tls:
+              mode: DISABLE
+        ---
+        EOF
+        gateway.networking.istio.io "prometheus-gateway" configured
+        virtualservice.networking.istio.io "prometheus-vs" configured
+        destinationrule.networking.istio.io "prometheus" configured
+        {{< /text >}}
+
+   1. Apply the tracing configuration.
+
+        {{< text bash >}}
+        $ cat <<EOF | kubectl apply -f -
+        apiVersion: networking.istio.io/v1alpha3
+        kind: Gateway
+        metadata:
+          name: tracing-gateway
+          namespace: istio-system
+        spec:
+          selector:
+            istio: ingressgateway
+          servers:
+          - port:
+              number: 15032
+              name: https-tracing
+              protocol: HTTPS
+            tls:
+              mode: SIMPLE
+              serverCertificate: sds
+              privateKey: sds
+              credentialName: telemetry-gw-cert
+            hosts:
+            - "$TELEMETRY_DOMAIN"
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: VirtualService
+        metadata:
+          name: tracing-vs
+          namespace: istio-system
+        spec:
+          hosts:
+          - "$TELEMETRY_DOMAIN"
+          gateways:
+          - tracing-gateway
+          http:
+          - match:
+            - port: 15032
+            route:
+            - destination:
+                host: tracing
+                port:
+                  number: 80
+        ---
+        apiVersion: networking.istio.io/v1alpha3
+        kind: DestinationRule
+        metadata:
+          name: tracing
+          namespace: istio-system
+        spec:
+          host: tracing
+          trafficPolicy:
+            tls:
+              mode: DISABLE
+        ---
+        EOF
+        gateway.networking.istio.io "tracing-gateway" configured
+        virtualservice.networking.istio.io "tracing-vs" configured
+        destinationrule.networking.istio.io "tracing" configured
+        {{< /text >}}
+
+1. Visit the telemetry add-ons via your browser.
+
+   * Kiali: `https://$TELEMERY_DOMAIN:15029/`
+   * Prometheus: `https://$TELEMERY_DOMAIN:15030/`
+   * Grafana: `https://$TELEMERY_DOMAIN:15031/`
+   * Tracing: `https://$TELEMERY_DOMAIN:15032/`
```

Comment:
in the above 4 lines, there is a typo in the env var name - its "TELEMETRY" not "TELEMERY"

Reply:
good catch.